### PR TITLE
feat(REST errors handling): Handle 401 and 503 error on Bonita REST requests

### PR DESCRIPTION
**Pull request by abirembaut**
* Add interceptor to reload the top level Bonita app page when there is a 401 or a 503 and the request if for Bonita REST API

Covers [DEV-485](https://bonitasoft.atlassian.net/browse/DEV-485)
Relates to [RUNTIME-531](https://bonitasoft.atlassian.net/browse/RUNTIME-531)

[DEV-485]: https://bonitasoft.atlassian.net/browse/DEV-485?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RUNTIME-531]: https://bonitasoft.atlassian.net/browse/RUNTIME-531?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by abirembaut** [comment]
@benjaminParisel I am currently trying

**Comment by abirembaut** [comment]
@benjaminParisel done

**Review by benjaminParisel — APPROVED** [review]
LGTM ! I think we can add translation for French and Spanish at least before merge, what did you think ? (and Ask for japanese translations)

**Comment by abirembaut** [comment]
yes. I though it was in crowdin but not for the localization.json, right ?

File: generator-angularjs/src/main/resources/templates/page/assets/json/localization.json
Comment (abirembaut) [inline generator-angularjs/src/main/resources/templates/page/assets/json/localization.json:187]: @pabloalonso could you validate that this translation is accurate, please ?

**Comment by abirembaut** [comment]
Waiting for the Japanese translation. Product team is going to contact the Japanese contributor.

**Comment by abirembaut** [comment]
@benjaminParisel since we don't know how long it is going to take to obtain the translations, maybe it can be merged and I will do another PR once I get them ? This way we can release a version an update the pages to validate the entire flow.

**Pull request merged**
